recipes-core/ncurses/ncurses_%.bbappend

```diff
@@ -0,0 +1 @@
+CACHED_CONFIGUREVARS_append_toolchain-clang = " cf_cv_prog_CC_c_o=yes cf_cv_prog_CXX_c_o=yes"
```

Comment:
> On May 17, 2016, at 9:58 AM, dankm <notifications@github.com> wrote:
> 
> In recipes-core/ncurses/ncurses_%.bbappend <https://github.com/kraj/meta-clang/pull/18#discussion_r63561490>:
> 
> > @@ -0,0 +1 @@
> > +CACHED_CONFIGUREVARS_append_toolchain-clang = " cf_cv_prog_CC_c_o=yes cf_cv_prog_CXX_c_o=yes"
> it is on hard-float arm. Anywhere else I've tested it works fine without the ncurses fix.
> 
> 

I build regularly for rpi3 and haven’t seen this issue. whats best way to reproduce it ?

> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/kraj/meta-clang/pull/18/files/734841aa35fa72e5536d675156838ee12f24a641..7c3e5f3e3e193f085b99e48d786a4826205f721d#r63561490>